ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -52,14 +73,14 @@ sub process_path {
     # Pull out the values and check that they really are defined.
     my $path = $pathentry->{path};
     if (!defined($path)) {
-        $self->error("entry with undefined path");
+        $self->error("Entry with undefined path");
         return 0;
     }
 
     # Get the owner.
     my $owner = $pathentry->{owner};
     if (!defined($owner)) {
-        $self->error("entry with undefined owner");
+        $self->error("Entry with undefined owner");
```

Comment:
hah, indeed. you could add the path to the message, but than that could be said about all messages. so feel free to do it or not, i'll stopping makeing remarks about code you didn't write